scripts/add-validator-balances.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const { promisify } = require('util');
+const readFile = promisify(fs.readFile);
+const assert = require('assert');
+async function main() {
+  let specFile = await readFile(__dirname + '/../pos-contracts/spec.json', 'UTF-8');
+  assert.ok(typeof specFile === 'string');
+  specFile = JSON.parse(specFile);
+  const accounts = specFile.accounts;
+  assert.ok(accounts != null && Object.getPrototypeOf(accounts) === Object.prototype);
+  for (const validator of process.env.INITIAL_VALIDATORS.split(',')) {
```

Comment:
Only the `OWNER` can have initial balance. Validators in general don't need balance.

Reply:
Actually we might not need any initial balance in any accounts since the owner is now white-listed to make 0-gas price transactions: https://github.com/poanetwork/pos-contracts/commit/bbf8bf7fa1973cf956327443236750c05074b0f0.